Formula/mariadb@10.7.rb

```diff
@@ -36,7 +24,9 @@ class MariadbAT107 < Formula
   keg_only :versioned_formula
 
   # See: https://mariadb.com/kb/en/changes-improvements-in-mariadb-107/
-  disable! date: "2023-02-01", because: :unsupported
+  # and https://mariadb.org/about/#maintenance-policy
+  # End-of-life on 2023-02-09. Disable date set to 3 months after.
+  disable! date: "2023-05-09", because: :unsupported
```

Comment:
Official EOL is 2023-02-09 from https://mariadb.org/about/#maintenance-policy

Release series | Stable (GA) Date | End-of-life
--- | --- | ---
[10.7](https://mariadb.com/kb/en/changes-improvements-in-mariadb-107/) | 9 Feb 2022 | 9 Feb 2023

In order to let users get final update before disabling, I extended disable date by 3 months.